src/components/Molecules/Dropdown/Dropdown.test.js

```diff
@@ -26,6 +26,7 @@ describe('<Dropdown />', () => {
       </Dropdown>
     );
     expect(wrapper.find('.dropdown')).toHaveLength(0);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Instead of adding a snapshot test to this open/close test, perhaps we should extend the Dropdown component in the snapshot test on line 34 so that it has children.